modules/mono/editor/GodotTools/GodotTools/Build/MSBuildPanel.cs

```diff
@@ -183,10 +183,14 @@ public override void _Notification(int what)
         {
             base._Notification(what);
 
-            if (what == NotificationThemeChanged) {
-                buildMenuBtn.Icon = GetThemeIcon("Play", "EditorIcons");
-                errorsBtn.Icon = GetThemeIcon("StatusError", "EditorIcons");
-                warningsBtn.Icon = GetThemeIcon("NodeWarning", "EditorIcons");
+            if (what == NotificationThemeChanged)
+            {
+                if (buildMenuBtn != null)
+                    buildMenuBtn.Icon = GetThemeIcon("Play", "EditorIcons");
+                if (errorsBtn != null)
+                    errorsBtn.Icon = GetThemeIcon("StatusError", "EditorIcons");
+                if (warningsBtn != null)
+                    warningsBtn.Icon = GetThemeIcon("NodeWarning", "EditorIcons");
```

Comment:
Null conditional operator `?.` would be shorter and more readable. But it's nothing important.

Reply:
I can't use it on an assignment: `The left-hand side of an assignment must be a variable, property or indexer`.